zerver/lib/zulip_update_announcements.py

```diff
@@ -0,0 +1,212 @@
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import List, Optional
+
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q, QuerySet
+from django.utils.timezone import now as timezone_now
+from django.utils.translation import override as override_language
+
+from zerver.actions.message_send import (
+    do_send_messages,
+    internal_prep_huddle_message,
+    internal_prep_stream_message,
+)
+from zerver.lib.message import SendMessageRequest
+from zerver.models.realm_audit_logs import RealmAuditLog
+from zerver.models.realms import Realm
+from zerver.models.users import UserProfile, get_system_bot
+
+
+@dataclass
+class ZulipUpdateAnnouncement:
+    level: int
+    message: str
+
+
+# We don't translate the announcement message because they are quite unlikely to be
+# translated during the time between when we draft them and when they are published.
+zulip_update_announcements: List[ZulipUpdateAnnouncement] = [
+    ZulipUpdateAnnouncement(
+        level=1,
+        message="""\
+This topic will receive [automated updates]({zulip_update_announcements_help_url}) \
+about new Zulip features.
+
```

Comment:
@timabbott and I chatted about this one, and are now thinking the org admin part can be skipped here. Maybe something like this?

To help you learn about improvements in the Zulip app, this topic will receive [automated updates]({zulip_update_announcements_help_url}) about new Zulip features. [Learn more](/help/configure-notification-bot#zulip-update-announcements).

Reply:
It might be worth making the help center link more explicit like this (and fleshing out the help center section more).

@timabbott what do you think?

Reply:
https://zulip.com/help/configure-notification-bot#zulip-update-announcements is the section to link to now.

We still need to migrate this help page to a new URL (/help/configure-automated-notices). @prakhar1144 maybe you could take care of that?

Reply:
Maybe something like this for the message text?

---

Zulip is introducing **Zulip updates**! To help you learn about new features and configuration options, this topic will receive messages about important changes in Zulip.

You can read update messages whenever it's convenient, or [mute](https://zulip.com/help/mute-a-topic) this topic if you are not interested. If you organization does not want to receive these announcements, they can be disabled. [Learn more](https://github.com/zulip/zulip/pull/%7Bzulip_update_announcements_help_url%7D).

---

Reply:
Yeah that works for me!

Reply:
<details><summary>Updated message</summary>
<img src="https://github.com/zulip/zulip/assets/56781761/7af1bb72-3f5a-45b7-9c35-4024ac00db6d">
</img>
</details>